packages/assistify-ai/server/lib/KnowledgeAdapterProvider.js

```diff
@@ -7,7 +7,7 @@ export function getKnowledgeAdapter() {
 	let knowledgeSource = '';
 
 	const KNOWLEDGE_SRC_APIAI = '0';
-	const KNOWLEDGE_SRC_SMARTI = '2';
+	const KNOWLEDGE_SRC_SMARTI = '1';
```

Comment:
Selecting the knowledge source by index is dependent on the order and defined sources within `config.js`. This should be solved in a more generic way.